### PR TITLE
Email for 10 last articles

### PR DESCRIPTION
**Pull request by IgnacioColuccio**
### Summary
- Added a button to send an e-mail containing the last 10 articles (by updated_at)
- Added configuration for actionmailer

### Trello card
https://trello.com/c/zJJm7TkI/14-crear-email-con-los-ultimos-10-articulos-creados

### Test plan
1. Login
2. Click on the "Send the latest articles to my e-mail" link on the navbar.
3. An HTML e-mail should be sent to the user's address.

### CODE REVIEW

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -17,21 +14,30 @@ html
           = current_user.email
           |  | 
           = link_to t(:signout), destroy_user_session_path, method: :delete
+          |  | 
+          = link_to t(:last_articles), last_articles_path, :method => :post
```
Comment (Dinuuu) [inline app/views/layouts/application.html.slim]: Try to use the new syntax, `method: :post`

**Comment by IgnacioColuccio** [comment]
Fixed

File: config/environments/development.rb [hunk 2]
```diff
@@ -40,4 +40,17 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+  config.action_mailer.smtp_settings = {
```
Comment (Dinuuu) [inline config/environments/development.rb]: Remember not to leave sensitive data here, use secret.yml and AppConfiguration

Comment (epintos) [inline config/environments/development.rb]: @IgnacioColuccio Usually sensitive data is like a problem in Rails. There is not conventions and lots of Gems that do the same. Some people gitignores the secrets.yml. the problem with that is that we hate sending this files by email between a team. So we push secrets.yml with no sensitive data. But what happens when we need sensitive data to try things in development? AppConfiguration is a good option (is developed by the awesome woloxer guido marucci), and there are some other alternatives like figaro, dotenv

**Comment by IgnacioColuccio** [comment]
Removed the environment variables, switched to appconfiguration. Credentials are now in .config.yml (not commited).

**Comment by Dinuuu** [comment]
:+1:

**Pull request merged**